index.html

```diff
@@ -0,0 +1,198 @@
+                 Bill Ready attended Harvard Business School.</p>
+        </div>
+
+       </div>
+
+    <div class="speakers-grid">
+        <img src="./img/redditCEO.jpg" class="speaker-image" alt="RedditCeo">
+        <div class="speaker-desc">
+            <h5>Steve Huffman</h5>
+            <h6>Reddit/CEO</h6>
+            <hr>
+            <p>Steve Huffman, also known by his Reddit username spez,
+                 is an American web developer and entrepreneur. 
+                 He is the co-founder and CEO of Reddit.</p>
+        </div>
+    </div>
+
+    <div class="speakers-grid">
+        <img src="./img/wordpressCEO.jpg" class="speaker-image" alt="Wordpressceo">
+        <div class="speaker-desc">
+            <h5>Matt Mullenweg</h5>
+            <h6>Wordpress/CEO</h6>
+            <hr>
+            <p>an American entrepreneur and web developer living in Houston.
+                 He is known for developing and founding the free and open-source web software WordPress,</p>
+        </div>
+    </div>
+
+    <div class="speakers-grid">
+        <img src="./img/Amazonceo.jpg" class="speaker-image" alt="Amazonceo">
+        <div class="speaker-desc">
+            <h5>Andy Jassy</h5>
+            <h6>Amazon/CEO</h6>
+            <hr>
+            <p>Andrew R. Jassy is an American business executive who is the president and chief executive officer of Amazon.</p>
+        </div>
+    </div>
+
+    <div class="speakers-grid">
+        <img src="./img/stripee.webp" class="speaker-image" alt="Stripeceo">
+        <div class="speaker-desc">
+            <h5>Patrick Collison</h5>
+            <h6>Stripe/CEO</h6>
+            <hr>
+            <p>Patrick Collison is an Irish billionaire entrepreneur. He is the co-founder and CEO of Stripe,
+                 which he started with his younger brother, John, in 2010. </p>
+        </div>
+    </div>
+
+    <div class="speakers-grid">
+        <img src="./img/paypalceo.webp" class="speaker-image" alt="Paypalceo">
+        <div class="speaker-desc">
+            <h5>Dan Schulman</h5>
+            <h6>Paypal/CEO</h6>
+            <hr>
+            <p>Daniel H. Schulman is an American business executive. He is president and CEO of PayPal,
+                 formerly serving as
+                  group president of enterprise growth at American Express.</p>
+        </div> -->
+     <!-- </div> -->
+    </div>
+ </section>
+ <footer>
+ <div class="partners">
+    <div class ="main-title">
+        <h2>Partners </h2>
+        <hr>
+    </div> 
+
+    <div class="companies">
+        <div class="brands"><i class="fa fa-amazon">Amazon</i></div>
+        <div class="brands"><i class="fa fa-google">Google</i></div>
+        <div class="brands"><i class="fa fa-facebook">Facebook</i></div>
+        <div class="brands"><i class="fa fa-twitter">Twitter</i></div>
+        <div class="brands"><i class="fa fa-cc-paypal">Paypal</i></div>
+    </div>
+</div>
+
+<div class="trademark">
```

Comment:
- [ ] Kindly fix the space between your logo the copyright is too much the same for the about page.

Reply:
@Kossi-stack what you mean by to much the same for about page?